agentlite/llm/agent_llms.py

```diff
@@ -50,6 +60,20 @@ def run(self, prompt: str):
         )
         return response.choices[0].message.content
 
+class VllmChatModel(BaseLLM):
+    def __init__(self, llm_config: LLMConfig):
+        super().__init__(llm_config)
+        self.client = OpenAI(base_url="http://localhost:8000/v1", api_key="EMPTY")
```

Comment:
This is the default endpoint in vLLM, but I could add it as an env variable if preferred